lua/cosmic/theme/plugins.lua

```diff
@@ -153,6 +154,15 @@ function M.init(use, config)
     end,
     disable = config.theme ~= 'github',
   })
+
+  use({
+    'tanvirtin/monokai.nvim',
+    as = 'monokai',
+    config = function()
```

Comment:
Can you also pin the current version of the theme?

I need to go around and do that for all of the themes.

Reply:
Thanks for the review. 

Currently the [monokai](https://github.com/tanvirtin/monokai.nvim) theme does not have versioning.
Do you want me to use the commit hash, instead of the version?

Reply:
done